packages/nextjs/src/common/_error.ts

```diff
@@ -53,7 +54,5 @@ export async function captureUnderscoreErrorException(contextOrProps: ContextOrP
     });
   });
 
-  // In case this is being run as part of a serverless function (as is the case with the server half of nextjs apps
-  // deployed to vercel), make sure the error gets sent to Sentry before the lambda exits.
-  await flushQueue();
+  vercelWaitUntil(flushSafelyWithTimeout());
```

Comment:
hmm, could we possibly still encapsulate this in some `flushQueue` util or so? Feels a bit "dirty" to have all these `vercelXXX` calls peppered throughout the code base, I think I'd prefer it if we could abstract this away a bit 🤔

Reply:
tbh it feels more dirty to me to mangle Vercel API with our own intentions. `vercelWaitUntil` is basically just the vendored `waitUntil`. Also, it's not like this is not readable or anything.

Reply:
We literally call `vercelWaitUntil(flushSafelyWithTimeout());` every time where we are calling this, as far as I see? 😅 

I think it would be nicer to still just call `flushQueue` everywhere (Which can literally be: `function flushQueue() { vercelWaitUntil(flushSafelyWithTimeout()); }` now), call this everywhere. if we need to adjust this logic (e.g. it does more or less or whatever) we can encapsulate this there, no need to repeat this everywhere - we call this 9 times now?

But no strong feelings, if you prefer it this way all good for me :)

Reply:
I don't see the upside so I'll leave it as is.